src/app/modules/user-teacher/teacher-post/components/teacher-create-post-form/teacher-create-post-form.component.ts

```diff
@@ -11,31 +11,37 @@ import { TeacherAddPost } from '../../store/actions';
 export class TeacherCreatePostFormComponent implements OnInit {
 
   createPostForm!: FormGroup;
-  
+
   constructor(
     private fb: FormBuilder,
     private store: Store
-    ) { }
+  ) { }
 
   ngOnInit() {
     this.initForm();
   }
 
-  initForm(){
+  initForm() {
     this.createPostForm = this.fb.group({
-     title: [''],
-     value: ['']
+      title: ['', [Validators.required, Validators.maxLength(255)]],
+      value: ['', [Validators.required, Validators.maxLength(255)]]
     });
-   }
+  }
 
   onSubmit() {
     const title = this.createPostForm.controls['title'].value
     const value = this.createPostForm.controls['value'].value
```

Comment:
Я думаю, когда настроим линтер, проверим весь код 🤓
```suggestion
    const value = this.createPostForm.controls['value'].value;
```